dashboard/config/routes.rb

```diff
@@ -285,6 +285,11 @@ def redirect_to_teacher_dashboard
     concerns :ops_routes
   end
 
+  get '/plc/content_creator/show_courses_and_modules', to: 'plc/content_creator#show_courses_and_modules'
+  ['courses', 'learning_modules', 'tasks', 'course_units', 'evaluation_questions'].each do |object|
+    get '/plc/' + object, to: redirect('plc/content_creator/show_courses_and_modules')
+  end
```

Comment:
I think this is okay for now.  But we should think through what we want the final link structure to be before release, when people will start bookmarking URLs and they become hard to change.